classes/log/store.php

```diff
@@ -53,12 +54,21 @@ class store extends php_obj implements log_writer {
 
     protected $loggingenabled = false;
 
+    /** @var bool $logguests true if logging guest access */
+    protected $logguests;
+
+    /** @var array $routes An array of routes to include */
+    protected $routes = array();
+
     /**
      * Constructs a new store.
      * @param log_manager $manager
      */
     public function __construct(log_manager $manager) {
         $this->helper_setup($manager);
+        $this->logguests = $this->get_config('logguests', 1);
```

Comment:
This too is copied from the Moodle core (database store) which already implements the guests' events filtering.